go/vt/sqlparser/ast_test.go

```diff
@@ -248,6 +248,14 @@ func TestDDL(t *testing.T) {
 			},
 		},
 		affected: []string{"a", "b"},
+	}, {
```

Comment:
Should add some simple tests to parse_test.go as well, with variants of these statements (including =, floats, etc)